web/satellite/src/components/project/DeleteProjectPopup.vue

```diff
@@ -104,26 +106,27 @@
                 return;
             }
 
-            this.isLoading = true;
-
             if (!this.validateProjectName()) {
                 return;
             }
 
-            if (!await this.deleteProject()) {
-                return;
-            }
+            try {
+                this.isLoading = true;
+                await this.$store.dispatch(PROJECTS_ACTIONS.DELETE, this.$store.getters.selectedProject.id);
```

Comment:
do we need to cover all that code in try block, or just ``store.dispatch`` action?